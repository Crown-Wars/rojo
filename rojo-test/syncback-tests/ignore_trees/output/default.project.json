{
  "name": "ignore_trees",
  "tree": {
    "$className": "DataModel",
    "ReplicatedStorage": {
      "$path": "src"
    }
  },
  "syncbackRules": {
    "ignoreTrees": [
      "ignore_trees/ReplicatedStorage/IgnoreMe"
    ]
  }
}