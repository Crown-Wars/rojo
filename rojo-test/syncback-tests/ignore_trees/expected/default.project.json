{
  "name": "ignore_trees",
  "tree": {
    "$className": "DataModel",
    "ReplicatedStorage": {
      "$path": "src"
    }
  },
  "syncbackRules": {
    "ignoreTrees": [
      "ReplicatedStorage/IgnoreMe"
    ]
  }
}